### PR TITLE
Use apiserver image which includes certs

### PR DESCRIPTION
**Pull request by MarcelMue**
Towards
https://github.com/giantswarm/giantswarm/issues/11946
https://github.com/giantswarm/giantswarm/issues/11707

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -20,7 +20,7 @@ require (
 	github.com/giantswarm/helmclient v1.0.3
 	github.com/giantswarm/ipam v0.2.0
 	github.com/giantswarm/k8sclient/v3 v3.1.0
-	github.com/giantswarm/k8scloudconfig/v6 v6.2.3
+	github.com/giantswarm/k8scloudconfig/v6 v6.2.7-0.20200623090521-bf10cd2570a1
```
Comment (MarcelMue) [inline go.mod:23]: TODO: Fix when k8scc is released.

**Comment by MarcelMue** [comment]
This is finally ready to merge IMO. I will test once again before tagging but clusters in china came up fine finally.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,18 +8,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
-- Add mapping between similar instance types `m4.16xlarge` and `m5.16xlarge`
+### Added
+
+- Add mapping between similar instance types `m4.16xlarge` and `m5.16xlarge`.
+- Add `lifecycle` label to the `aws_operator_ec2_instance_status` metric to distinguish on-demand and spot.
+
+### Changed
+
+- Use `k8s-apiserver` image which includes certs.
```
Comment (tfussell) [inline CHANGELOG.md:18]:
```suggestion
- Use `k8s-apiserver` image which includes CAs to enable OIDC.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,18 +8,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- Add mapping between similar instance types `m4.16xlarge` and `m5.16xlarge`
+### Added
+
+- Add mapping between similar instance types `m4.16xlarge` and `m5.16xlarge`.
+- Add `lifecycle` label to the `aws_operator_ec2_instance_status` metric to distinguish on-demand and spot.
```
Comment (tfussell) [inline CHANGELOG.md:14]: Are these incidental changes from upgrading k8scloudconfig?

Comment (MarcelMue) [reply CHANGELOG.md:14]: I just moved this one into the correct section - the changelog was not created by me.

Comment (tfussell) [reply CHANGELOG.md:14]: Ah, missed that.

**Pull request merged**